tests/app/validation/test_number_range_validator.py

```diff
@@ -195,6 +195,51 @@ def test_manual_max(self):
         except ValidationError:
             self.fail("Valid integer raised ValidationError")
 
+
+    def test_zero_max(self):
+        answer = { 'max_value': {
+                        "value": 0
+                    },
+                   'label': 'Max Test',
+                   'guidance': '',
+                   'options': [],
+                   'mandatory': False,
+                   'validation':
```

Comment:
You don't really need to define this here, it just bloats the tests when you could just include `error_messages.py`

Reply:
Done